plugins/workspace-search/src/workspace_search.ts

```diff
@@ -156,7 +156,9 @@ export class WorkspaceSearch implements Blockly.IPositionable {
     this.addEvent(this.inputElement, 'input', this, () => this.onInput());
     this.addEvent(this.inputElement, 'click', this, () => {
       this.searchAndHighlight(this.searchText, this.preserveSelected);
-      this.inputElement.select();
+      if (this.inputElement) {
+        this.inputElement.select();
+      }
```

Comment:
I think you can use an optional chaining operator here.
```suggestion
      this.inputElement?.select();
```